get-started/index.md

```diff
@@ -1,67 +1,54 @@
- *  [Architectural Principles](/nservicebus/architecture/principles.md) and [Bus vs Broker architecture](/nservicebus/architecture/): A high-level overview of messaging architectures and the two popular approaches to building them.
- * [Concepts Overview](/nservicebus/concept-overview.md): Descriptions of the concepts, features and vernacular of NServiceBus.
- * [Hosting](/nservicebus/hosting/): Provides information on the various approaches to hosting an instance of NServiceBus.
 
+```ps
+PM> Install-Package NServiceBus
+```
 
-## Tutorials
-
-The [NServiceBus Quick Start](/tutorials/quickstart/) tutorial gives a quick overview of the benefits of using NServiceBus, and shows how software systems built on asynchronous messaging using NServiceBus are superior to traditional synchronous HTTP-based web services.
-
-The [Introduction to NServiceBus](/tutorials/intro-to-nservicebus/) tutorial gives a guided walk-through of building an NServiceBus system and introduces the basic concepts of messaging. Each lesson includes a downloadable solution with the completed exercise.
-
-The [Scaling Applications with Microservices and NServiceBus 6](https://www.pluralsight.com/courses/microservices-nservicebus6-scaling-applications) Pluralsight course provides guidance on MicroServices and how to build them using NServiceBus.
-
-The [Monitoring NServiceBus](/tutorials/monitoring-demo/) tutorial is the best way to get familiar with the monitoring features in the Particular Service Platform before deploying monitoring on a real system. The included downloadable sample contains all of the necessary parts of the platform, already configured and ready to run.
-
-
-## Samples
-
-Samples are a great way to learn and explore NServiceBus by looking at runnable code. They are fully functional solutions that can be run from Visual Studio. Learn by exploring code. Each sample includes an explanation of the scenario and the technology or concept that it illustrates.
-
-Go to [Introduction to Sample](/samples/) or [Skip to the list of samples](/samples/#related-samples).
-
-The following samples are recommended for getting started:
-
- * [Step by Step Sample](/samples/step-by-step/): Illustrates essential NServiceBus concepts by showing how to build a simple system.
- * [Endpoint configuration choices](/samples/endpoint-configuration/): Walks through the most common choices required when creating a first endpoint. It will also show the configuration APIs needed to implement those choices.
- * [On Premise Show Case](/samples/show-case/on-premise/): An implementation of a fictional store that shows many features of NServiceBus working together.
 
+### Sending a message
 
-## Downloading
+```cs
+await endpoint.Send(new SubmitOrder { OrderId = orderId });
```

Comment:
Tutorials don't use `.configureawait(false)`?

Reply:
Tutorials do, but we don't need that boilerplate in a "look how easy the API is", right?